### PR TITLE
[14.0] mig: account_vat_period_end_statement

### PR DESCRIPTION
**Pull request by fredzamoabg**
account_vat_period_end_statement module migration from 12.0

### CODE REVIEW

**Comment by eLBati** [comment]
Eventualmente da includere: https://github.com/OCA/l10n-italy/pull/2218

**Comment by eLBati** [comment]
Ecco https://github.com/OCA/l10n-italy/issues/2221

File: account_vat_period_end_statement/migrations/14.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#  Copyright 2021 Alfredo Zamora - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr, "account_credit_control", "migrations/14.0.1.0.0/noupdate_changes.xml"
```
Comment (SimoRubi) [inline account_vat_period_end_statement/migrations/14.0.1.0.0/post-migration.py:10]:
```suggestion
        env.cr, "account_vat_period_end_statement", "migrations/14.0.1.0.0/noupdate_changes.xml"
```

Comment (fredzamoabg) [reply account_vat_period_end_statement/migrations/14.0.1.0.0/post-migration.py]: fissato grazie.

**Comment by eLBati** [comment]
Moving to https://github.com/OCA/l10n-italy/pull/2230

**Pull request closed without merging**